src/cdk-experimental/popover-edit/edit-ref.ts

```diff
@@ -51,11 +52,10 @@ export class EditRef<FormValue> implements OnDestroy {
    * applicable.
    */
   init(previousFormValue: FormValue|undefined): void {
-    // Wait for either the first value to be set, then override it with
-    // the previously entered value, if any.
-    this._form.valueChanges!.pipe(take(1)).subscribe(() => {
+    // Wait for the zone to stabilize before caching the initial value.
+    // This ensures that all form controls have been initialized.
+    this._ngZone.onStable.asObservable().pipe(take(1)).subscribe(() => {
```

Comment:
The call to asObservable is redundant - pipe(take(1)) will return an Observable (also we're not exposing this as a publicly so even using the Subject directly is not risky in this case)